definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState.json

```diff
@@ -2259,5 +2259,299 @@
     "CaseStateID": "DNisRefused",
     "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "15/06/2020",
```

Comment:
`LiveFrom` is ignored, but just to make it easier to understand when things were added I'd suggest to use date of release.

As I said - it's completely ignored by CCD.

Reply:
Resolved using Mondays go live date for the SS 14/09/2020